.github/workflows/ci.yaml

```diff
@@ -152,6 +163,17 @@ jobs:
           # FIXME: It seems some items in `src/unix/mod.rs`
           # aren't defined on redox actually.
           # - x86_64-unknown-redox
+        include:
+          - target: arm-unknown-linux-gnueabihf
+          - target: arm-unknown-linux-gnueabihf
+            env:
+              RUST_LIBC_UNSTABLE_GNU_FILE_OFFSET_BITS64: 1
+            artifact-tag: offset-bits64
+          # - target: powerpc-unknown-linux-gnu
+          # - target: powerpc-unknown-linux-gnu
```

Comment:
Could you keep the `FIXME(ppc): SIGILL running tests ...` note?

Reply:
Oops, I meant to include that.